### DISCUSSION TITLE
Changing the order of the front pages and abstract content

### DISCUSSION THREAD

**Discussion by PublicHealthDataGeek** [nmfs-opensci/quarto-thesis#14] [Q&A] [answered]
Hello
I wish to change the order of the front pages to have them in this order:
Dedication
Acknowlegements
Abstract.
But when I tried to do this by moving the below code in the `before-body.tex` to after the equivalent code for acknowledgements, it didnt seem to work.  

```
$if(abstract)$
%----------------------------------------------------------------------------------------
%	ABSTRACT PAGE
%----------------------------------------------------------------------------------------

\begin{abstract}
\addchaptertocentry{\abstractname} % Add the abstract to the table of contents
$abstract$
\end{abstract}

$endif$
```
Also I want to simplify the abstract page so it just has the title (abstract) and then my abstract text.  Looking at the MasterDoctoralThesis.cls  from lines 325- 394 control the abstract.  Can I just delete this and copy the code from something like the Dedication section instead?  For me it would be simpler for it to be structured in the same way as the other front matter.

Any help or advice gratefully received.

**Comment by PublicHealthDataGeek** [comment, marked as answer]
OK I figured this out.  To make any changes to the layout the MasterDoctoralThesis.cls in the _extensions folder needs to be changed and saved.  You cant alter the MastersDoctoralThesis.cls in the main folder as a) this is read only and b) it isnt the one that controls the layout!  

To solve the issue, I commented out the lines I didnt want to be rendered in the cls in the _extensions folder and then it worked fine.

**Comment by eeholmes** [comment]
Perfect! Glad you figured it out. So what happens when you use extensions is that it copies the helper files it needs from the _extensions folder into your main folder. I will add some notes on the README so that future users know how to customize the thesis class.
